specification/confluent/resource-manager/Microsoft.Confluent/preview/2020-03-01-preview/confluent.json

```diff
@@ -0,0 +1,697 @@
+      "title": "ErrorResponseBody",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code",
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "description": "Error message",
+          "type": "string",
+          "readOnly": true
+        },
+        "target": {
+          "description": "Error target",
+          "type": "string",
+          "readOnly": true
+        },
+        "details": {
+          "description": "Error detail",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ErrorResponseBody"
+          }
+        }
+      }
+    },
+    "ResourceProviderDefaultErrorResponse": {
+      "type": "object",
+      "description": "Default error response for resource provider",
+      "title": "ResourceProviderDefaultErrorResponse",
+      "properties": {
+        "error": {
+          "description": "Response body of Error",
+          "readOnly": true,
+          "$ref": "#/definitions/ErrorResponseBody"
+        }
+      }
+    },
+    "ProvisioningState": {
+      "enum": [
+        "Accepted",
+        "Creating",
+        "Updating",
+        "Deleting",
+        "Succeeded",
+        "Failed",
+        "Canceled"
+      ],
+      "type": "string",
+      "description": "Provision states for confluent RP",
+      "title": "ProvisioningState",
+      "x-ms-enum": {
+        "name": "ProvisionState",
+        "modelAsString": true
+      }
+    },
+    "OfferDetail": {
+      "description": "Confluent Offer detail",
+      "type": "object",
+      "properties": {
+        "publisherId": {
+          "description": "Publisher Id",
+          "maxLength": 50,
+          "type": "string"
+        },
+        "id": {
+          "description": "Offer Id",
+          "maxLength": 50,
+          "type": "string"
+        },
+        "planId": {
+          "description": "Offer Plan Id",
+          "maxLength": 50,
+          "type": "string"
+        },
+        "planName": {
+          "description": "Offer Plan Name",
+          "maxLength": 50,
+          "type": "string"
+        },
+        "termUnit": {
+          "description": "Offer Plan Term unit",
+          "maxLength": 25,
+          "type": "string"
+        }
+      }
+    },
+    "UserDetail": {
+      "description": "Subscriber detail",
+      "type": "object",
+      "properties": {
+        "firstName": {
+          "description": "First name",
+          "maxLength": 50,
+          "type": "string"
+        },
+        "lastName": {
+          "description": "Last name",
+          "maxLength": 50,
+          "type": "string"
+        },
+        "emailAddress": {
+          "description": "Email address",
+          "pattern": "^\\w+@[a-zA-Z_]+?\\.[a-zA-Z]{2,3}$",
+          "type": "string"
+        },
+        "phoneNumber": {
+          "description": "Phone number",
```

Comment:
it will be great to put string pattern  in description for user refernece.

Reply:
updated description with string pattern,